src/data_management/rasters/Raster.js

```diff
@@ -138,12 +162,13 @@ class Raster extends Component {
         };
         fetch(url + toBeDeletedRasterUuidsArray[i] + "/", opts);
         // Refresh the page, so that the removed rasters are no longer visible
-        this.setState({
-          checkAllCheckBoxes: false,
-          checkboxAllCheckboxesChecked: false
-        });
-        this.checkAllCheckBoxes(false);
-        this.loadRastersOnPage(this.state.page, this.state.searchTerms);
+        // fetch is a asynchrounous action. the following lines should only be executed on .then. todo fix this
+        // these lines are commented out because they also happen in getRastersFromApi
```

Comment:
`// Setting the checkboxes state is done in getRastersFromApi` 
(Kunnen de regels eronder ook weg.)